heliopy/data/imp.py

```diff
@@ -54,6 +56,26 @@ def merged(probe, starttime, endtime, try_download=True):
     dirs = []
     fnames = []
     extension = '.asc'
+    units = OrderedDict([('sw_flag', u.dimensionless_unscaled),
```

Comment:
I'm a bit confused about FCp and DWp.

Reply:
I'm not entirely sure what they are, but I know they are dimensionless.

Reply:
I've added dimensionless unit to that now!